lib/validators/specValidator.js

```diff
@@ -603,22 +596,35 @@ class SpecValidator {
       }
       let location = parameter.in;
       if (location === 'path' || location === 'query') {
+        if (location === 'path' && parameterValue && typeof parameterValue.valueOf() === "string") {
+          if (options.pathTemplate.charAt(options.pathTemplate.indexOf(`${parameter.name}`) - 2) === '/' && parameterValue.startsWith('/')) {
```

Comment:
what's "-2" looking for here? it'd be great to add a small comment potentially with an example on why we're adding this condition. There's a message you're constructing right after, but a comment wouldn't hurt for readability.